### PR TITLE
Setup Repo

### PR DESCRIPTION
**Pull request by emilyrohrbough**
# Summary
The browsers defined in `index.js` were scoped from the comments made on https://github.com/cerner/terra-toolkit/pull/101.

### CODE REVIEW

**Review by bjankord — APPROVED** [review]

**Comment by yuderekyu** [comment]
we should add a .gitignore

**Comment by emilyrohrbough** [comment]
@yuderekyu Matt set up the .gitignore with the repo creation https://github.com/cerner/browserslist-config-terra/blob/master/.gitignore

File: README.md [hunk 1]
```diff
@@ -1,2 +1,74 @@
...
+<p align="center">
+  <img height="128" width="128" src="https://github.com/cerner/browserslist-config-terra/raw/master/terra.png">
+</p>
+
+<!-- Name -->
+<h1 align="center">
+  Terra Browserslist Shared Config
+</h1>
+
+[![NPM version](http://img.shields.io/npm/v/terra-toolkit.svg)](https://www.npmjs.org/package/browserslist-config-terra)
```
Comment (mjhenkes) [inline README.md]: terra-toolkit.svg should probably be browserlist-config-terra.svg

File: scripts/release/release.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
+const shell = require('shelljs');
+
+const releaseType = process.argv[2] || 'patch';
+
+if (shell.exec(`npm run compile && npm run lint && npm version ${releaseType} -m "Released version %s" && npm publish && git push`).code !== 0) {
```
Comment (mjhenkes) [inline scripts/release/release.js]: The compile and lint commands will fail.

**Comment by mjhenkes** [comment]
Lets setup lint. I know if feels like overkill but i'd like to run something for CI.

**Comment by emilyrohrbough** [comment]
@mjhenkes I have linting set up. It won't show on this PR unless we break out that work into a new PR that is merged first. Let me know which you prefer.

**Review by mjhenkes — APPROVED** [review]
I'll trust you that lint is working, this time. :)

**Pull request merged**